distributor-api-v1/changelog.md

```diff
@@ -1,8 +1,13 @@
 # Changelog
 
+## 19.11.2020
+
+* Made `ConfigurationId` parameter in [CreateReservationGroup](operations.md#create-reservation-group) endpoint required.
+* Added `ConfigurationId` parameter to [GetAvailability](operations.md#get-availability) endpoint.
```

Comment:
I would add a note here as well that the previous variant of sending `HotelId` is deprecated

Reply:
Does writing it like this sound alright to you?:

>* Previous variant of sending only `HotelId` without `ConfigurationId` to these endpoints is deprecated.

 Right now we still need to send both `HotelId` and `ConfigurationId`. The api doesn't yet work with only `ConfigurationId`.
If you omit `HotelId` from the request, the api will respond with message `Invalid HotelId.`

Reply:
In the end I've added this: 
> Previous variant of sending `HotelId` without `ConfigurationId` to these endpoints was deprecated.

If it's not alright, please let me know and I'll reformulate it as followup.